functions-and-scope.js

```diff
@@ -82,3 +140,27 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // highestGrade(grades) geeft 9
 // highestGrade([6, 4, 5]) geeft 6
 // highestGrade([8, 9, 4, 6, 10]) geeft 10
+
+function returnHighestGrade(grades) {
+    return Math.max(...grades);
+}
+
+console.log(returnHighestGrade(grades));
+console.log(returnHighestGrade([6, 4, 5]));
+console.log(returnHighestGrade([8, 9, 4, 6, 10]));
+
+function returnHighestGrade2(grades) {
+    let highestGrade = 0;
```

Comment:
Het is altijd beter om te beginnen met de eerste waarde van de array. Beetje flauw bij dit voorbeeld, maar stel dat je een array hebt van `[-2, -5, -1, -3]`. Als nu deze array aan jouw functie mee zou geven, dan komt daar `0` uit, terwijl dat cijfer helemaal niet in de array voorkomt.

```suggestion
    let highestGrade = grades[0];
```